src/main/java/salesforce/ui/pages/product/newproduct/AbstractNewProductPage.java

```diff
@@ -60,6 +74,8 @@ public AbstractProductPage setProductInformation(final Product product, final Se
     protected HashMap<String, Runnable> composeStrategyMap(final Product product) {
         HashMap<String, Runnable> strategyMap = new HashMap<>();
         strategyMap.put(ProductConstant.NAME, () -> setProductName(product.getName()));
+        strategyMap.put(ProductConstant.CODE, () -> setProductCode(product.getCode()));
+        strategyMap.put(ProductConstant.DESCRIPTION, () -> setProductDescription(product.getDescription()));
```

Comment:
this I have already done

Reply:
I think, but I needed to run the scenario